### PR TITLE
test(@angular/cli): disable sourcemaps by default in test suite

### PR DESCRIPTION
**Pull request by filipesilva**
Note: this PR is for the repository test suite, not for project commands like `ng test`.

### CODE REVIEW

File: tests/e2e/utils/process.ts [hunk 1]
```diff
@@ -27,6 +27,15 @@ function _exec(options: ExecOptions, cmd: string, args: string[]): Promise<Proce
     `==========================================================================================`
   );
 
+  // Auto-add some flags to ng commands that build or test the app.
+  // --no-progress disables progress logging, which in CI logs thousands of lines.
+  // --no-sourcemaps disables sourcemaps, making builds faster.
+  // We add these flags before other args so that they can be overriden.
+  // e.g. `--no-sourcemaps --sourcemaps` will still generate sourcemaps.
+  if (cmd === 'ng' && ['build', 'serve', 'test', 'e2e', 'xi18n'].indexOf(args[0]) !== -1) {
+    args = [args[0], '--no-progress', '--no-sourcemaps', ...args.slice(1)]
```
Comment (hansl) [inline tests/e2e/utils/process.ts]: Feels like we should have default values for those in the config, but I cannot find them. Ideas?

Comment (filipesilva) [inline tests/e2e/utils/process.ts]: We don't have those in the `.angular-cli.json`, no. I could add support for it and set them on the project creation phase.

**Comment by filipesilva** [comment]
Blocked by https://github.com/angular/angular-cli/pull/6889, which adds build defaults support to `.angular-cli.json`. These defaults can then be set in the test project.

File: tests/e2e/utils/process.ts (hunk 1 above)
Comment (filipesilva) [inline tests/e2e/utils/process.ts]: Fixed now that https://github.com/angular/angular-cli/pull/6889 is in.

**Pull request merged**